### PR TITLE
Document options

### PR DESCRIPTION
**Pull request by olivroy**
Follow-up to #848., where support for `maxWidth` and `minWidth` was added.

Related to #20

### CODE REVIEW

File: R/class-comment.R [hunk 1]
```diff
@@ -118,7 +118,7 @@ wb_comment <- function(text = NULL,
                        width = 2,
                        height = 4) {
   # Code copied from the wbWorkbook
-  author <- author %||% Sys.getenv("USERNAME")
+  author <- author %||% Sys.getenv("USERNAME") %||% Sys.getenv("USER")
```
Comment (olivroy) [inline R/class-comment.R:121]: Oops, I just remember that `Sys.getenv()` returns "" when it can't find something. This will not work as intended.

Comment (JanMarvin) [reply R/class-comment.R:121]: Can we grep for USER in the environment? Is it an issue with Windows? I tested with Linux and Mac and afaik they all had USERNAME

Comment (olivroy) [reply R/class-comment.R:121]: For me (Windows) `Sys.getenv("USER")` returns `""` and `Sys.getenv("USERNAME")` works as expected. Let's drop "USER" then?

Comment (JanMarvin) [reply R/class-comment.R:121]: 👍 you introduced it, you can remove it

Comment (JanMarvin) [reply R/class-comment.R]: Darn! I lied. Sorry for the confusion :-(
My Mac shows
```R
> Sys.getenv("USERNAME")
[1] ""
> Sys.getenv("USER")
[1] "janmarvingarbuszus"
```

maybe something like:

```R
ifelse(Sys.getenv("USERNAME") == "", Sys.getenv("USER"), Sys.getenv("USERNAME"))
```

Comment (olivroy) [reply R/class-comment.R]: Thanks for validating. I will reuse the `unset` argument of `Sys.getenv()` instead of ifelse.

**Comment by JanMarvin** [comment]
Thanks, just one more thing, could you add `minWidth` to the code? Guess next to the `if widths > getOption(maxWidth)` part we might want something like: `if widths < getOption("minWidth")`.

**Comment by JanMarvin** [comment]
Thanks, merged!

**Pull request merged**